### PR TITLE
git-quick-stats 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by arzzen**
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Allow user to install [git-quick-stats](https://github.com/arzzen/git-quick-stats/)

### CODE REVIEW

File: Formula/git-quick-stats.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://github.com/arzzen/git-quick-stats"
+  url "https://github.com/arzzen/git-quick-stats/archive/1.0.1.tar.gz"
+  sha256 "397fabda3e894bf058767e32114ce8496ecf23f088e6de6203cae4486e1f755b"
+
+  def install
+    bin.install "git-quick-stats"
+  end
+
+  test do
+    system "false"
```
Comment (dunn) [inline Formula/git-quick-stats.rb]: You'll need a test here, please!

Comment (arzzen) [inline Formula/git-quick-stats.rb]: i fixed it

File: Formula/git-quick-stats.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://github.com/arzzen/git-quick-stats"
+  url "https://github.com/arzzen/git-quick-stats/archive/1.0.1.tar.gz"
+  sha256 "397fabda3e894bf058767e32114ce8496ecf23f088e6de6203cae4486e1f755b"
+
+  def install
+    bin.install "git-quick-stats"
+  end
+
+  test do
+    assert File.exist?("#{bin}/git-quick-stats")
```
Comment (dunn) [inline Formula/git-quick-stats.rb]: You can create a git repository in the temporary directory that the test runs in; see e.g., [`blackbox.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/blackbox.rb)

Comment (arzzen) [inline Formula/git-quick-stats.rb]: #dunn updated

File: Formula/git-quick-stats.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  url "https://github.com/arzzen/git-quick-stats/archive/1.0.1.tar.gz"
+  sha256 "397fabda3e894bf058767e32114ce8496ecf23f088e6de6203cae4486e1f755b"
+
+  def install
+    bin.install "git-quick-stats"
+  end
+
+  test do
+    system "git", "init"
+    assert File.exist?("#{bin}/git-quick-stats")
```
Comment (MikeMcQuaid) [inline Formula/git-quick-stats.rb]: Can remove this line.

File: Formula/git-quick-stats.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    assert_match /All branches \(sorted by most recent commit\)/, shell_output("#{bin}/git-quick-stats branchesByDate").strip
```
Comment (MikeMcQuaid) [inline Formula/git-quick-stats.rb]: Make the first argument a string so `(` doesn't need escaped. A newline before `shell_output` so it's <= 80 characters. Skip the `.strip`

File: Formula/git-quick-stats.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    assert_match /^Invalid argument/, shell_output("#{bin}/git-quick-stats command").strip
```
Comment (MikeMcQuaid) [inline Formula/git-quick-stats.rb]: Remove the `.strip`.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/git-quick-stats.rb (hunk 5 above)
Comment (arzzen) [inline Formula/git-quick-stats.rb]: @MikeMcQuaid done

File: Formula/git-quick-stats.rb (hunk 4 above)
Comment (arzzen) [inline Formula/git-quick-stats.rb]: done

File: Formula/git-quick-stats.rb (hunk 3 above)
Comment (arzzen) [inline Formula/git-quick-stats.rb]: done

**Comment by arzzen** [comment]
@MikeMcQuaid done

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**